packages/snip20/src/handle.rs

```diff
@@ -0,0 +1,562 @@
+use serde::Serialize;
+
+use cosmwasm_std::{to_binary, Binary, Coin, CosmosMsg, HumanAddr, StdResult, Uint128, WasmMsg};
+
+use secret_toolkit_utils::space_pad;
+
+/// SNIP20 token handle messages
+#[derive(Serialize)]
+#[serde(rename_all = "snake_case")]
+pub enum HandleMsg<'a> {
+    // Native coin interactions
+    Redeem {
+        amount: Uint128,
+        // TO DO: remove skip_serializing once denom is added to sSCRT stored on mainnet
```

Comment:
super petty thing - in Jetbrains products, comments that start with `todo` or `TODO` with no spaces, are highlighted and listed in a special section.
you definitely don't have to change this, but keep it in mind if you want the to-do to have a higher visibility :)